src/app/pages/market/market.twig

```diff
@@ -64,20 +64,22 @@
     <header>
       <h1>Axon Ivy Market</h1>
     </header>
+    
+    
     <p>
       The Axon Ivy Market offers a unique experience to accelerate process automation.
       From standard connectors of leading providers, ready-to-use process models,
       business solutions and many utilities such as document generation and workflow user interfaces.
     </p>
 
+  
+
     <div class="row">
       <div class="6u 12u(small)" style="margin-bottom:10px;">
         <span class="dropdown">
           <label>Topic</label>
           <select id="tags" multiple="multiple">
-            {% for tag in tags %}
```

Comment:
Isn't this one the topic/tag select filter? Why did you remove this one?

Reply:
ups